uint64 error_count
float32 x
float32 y
float32 z
float32 range_ga
float32 scaling
float32 temperature

int16 x_raw
int16 y_raw
int16 z_raw

uint32 device_id	# unique device ID for the sensor that does not change between power cycles
